Countries/FirstScreen/CountryTableViewCell.swift

```diff
@@ -52,18 +61,28 @@ class CountryTableViewCell: UITableViewCell {
     lazy private var nameLabel: UILabel = {
         let label = UILabel()
         label.textAlignment = .left
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.font = UIFont.systemFont(ofSize: Constants.fontSize)
         return label
     }()
     
     lazy private var capitalLabel: UILabel = {
         let label = UILabel()
         label.textAlignment = .left
+        label.numberOfLines = 0
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.systemFont(ofSize: Constants.fontSize)
+        label.lineBreakMode = .byTruncatingTail
         return label
     }()
     
     lazy private var regionLabel: UILabel = {
         let label = UILabel()
         label.textAlignment = .left
+        label.numberOfLines = 0
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.systemFont(ofSize: Constants.fontSize)
```

Comment:
why capitalLabel have lineBreakMode and others dont? or vise versa

Reply:
thank you, I just tested it and forgot to delete